app/src/main/java/ch/epfl/sdp/blindwar/database/UserDatabase.kt

```diff
@@ -216,4 +218,12 @@ object UserDatabase {
             }
         }
     }
+
+    /**
+     * Get current authenticated user
+     *
+     */
+    fun getCurrentUser(): DataSnapshot {

```

Comment:
I don't think that's the best way to get the current user since you pause the program until the future is completed.

Reply:
I also made a getCurrentUser so I will keep the necessary one